### PR TITLE
Update .md and more files

### PR DESCRIPTION
**Pull request by skanda890**
- [X] I have signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs).
- [X] This pull request is related to #4635
-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-cli/pull/4619)

### CODE REVIEW

**Comment by skanda890** [comment]
If this has any problem look at #4618

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 ![winget install wingetcreate](.github/images/WingetInstall.gif)
 
-If you are new to the Windows Package Manager, you might want to [Explore the Windows Package Manager tool](https://docs.microsoft.com/learn/modules/explore-windows-package-manager-tool/?WT.mc_id=AZ-MVP-5004737). The client has access to packages from two default sources. The first is "msstore" the Microsoft Store (free Apps rated "e" for everyone). The second is "winget" the [WinGet community repository](https://github.com/microsoft/winget-pkgs).
+If you are new to the Windows Package Manager, you might want to [Explore the Windows Package Manager tool](https://docs.microsoft.com/learn/modules/explore-windows-package-manager-tool/?WT.mc_id=AZ-MVP-5004737). The client has access to packages from two default sources. The first is "msstore" the Microsoft Store (free Apps rated "e" for everyone). The second is "WinGet" the [WinGet community repository](https://github.com/microsoft/winget-pkgs).
```
Comment (denelon) [inline README.md:7]:
```suggestion
If you are new to the Windows Package Manager, you might want to [Explore the Windows Package Manager tool](https://docs.microsoft.com/learn/modules/explore-windows-package-manager-tool/?WT.mc_id=AZ-MVP-5004737). The client has access to packages from two default sources. The first is "msstore" the Microsoft Store (free Apps rated "e" for everyone). The second is "winget" the [WinGet community repository](https://github.com/microsoft/winget-pkgs).
```

Comment (denelon) [reply README.md:7]: The WinGet source is "winget". You can use `winget source list` to see the casing for that one.

**Comment by skanda890** [comment]
Done.

**Comment by denelon** [comment]
Thanks! I'm good with the README.md, and we'll need one of the engineers to confirm the project files.

**Comment by skanda890** [comment]
> Thanks! I'm good with the README.md, and we'll need one of the engineers to confirm the project files.

@denelon, please tell any engineers to review this.

**Comment by skanda890** [comment]
> # @check-spelling-bot Report
> ### 🔴 Please review
> See the [📂 files](4619/files/) view or the [📜action log](https://github.com/microsoft/winget-cli/actions/runs/9925839383/job/27418708273#step:4:1) for details.
> 
> Unrecognized words (126)
> Previously acknowledged words that are now absent
> Some files were automatically ignored
> To accept ✔️ these unrecognized words as correct and remove the previously acknowledged and now absent words, run the following commands
> Available 📚 dictionaries could cover words not in the 📘 dictionary
> Errors (2)
> If the flagged items are 🤯 false positives

There is a bug while accepting suggestions.

File: .github/ISSUE_TEMPLATE/Feature_Request.yml
Comment (Trenly) [inline .github/ISSUE_TEMPLATE/Feature_Request.yml:1]: These spaces are intentional for readability

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (Trenly) [inline .github/PULL_REQUEST_TEMPLATE.md:1]: This newline is intentional to make it easier for contributors to add comments when opening a PR

File: .github/fabricbot.json
Comment (Trenly) [inline .github/fabricbot.json:1]: This file no longer needs to be in the repo. The fact that it is included in this PR makes me think your branch of `master` was outdated before you made updates

File: azure-pipelines.nuget.yml [hunk 2]
```diff
@@ -1,213 +1,213 @@
...
 parameters:
-  - name: version
-    displayName: Version to stamp on binaries and nuget package. Should be in form of "major.minor", for example "1.0"
-    type: string
-
-pool:
-  vmImage: "windows-latest"
+- name: version
+  displayName: Version to stamp on binaries and nuget package. Should be in form of "major.minor", for example "1.0"
+  type: string
```
Comment (Trenly) [inline azure-pipelines.nuget.yml:6]: Why are you removing the pool? I think you need to sync your branch with the latest changes from master

**Review by Trenly — CHANGES_REQUESTED** [review]
Most of your 154 commits appear to be outdated or duplicate of other changes that were pulled in many months ago. You likely need to rebase your branch

**Comment by skanda890** [comment]
How can I rebase my branch?

**Comment by Trenly** [comment]
> How can I rebase my branch?

It depends on your specific setup. If you have a git remote set up to track this repo then it would be `git rebase <remote_name>/master`, and then resolving all the conflicts. 

If there's too many merge conflicts, then it may be easier to reset your branch and either cherry-pick or redo your changes

**Comment by skanda890** [comment]
When I was trying to rebase the branch this happened:
image
image
(available in  )

**Comment by skanda890** [comment]
In the process of rebasing.

**Comment by skanda890** [comment]
Will rebase it by tomorrow 4:00 pm IST.

**Comment by skanda890** [comment]
Stop the @check-spelling bot.

**Review by skanda890 — COMMENTED** [review]
For me these changes look good but, @denelon, take a look and tell the engineers to also take a look.

**Comment by skanda890** [comment]
> Most of your 154 commits appear to be outdated or duplicate of other changes that were pulled in many months ago. You likely need to rebase your branch

Rebased!

**Comment by Trenly** [comment]
> > Most of your 154 commits appear to be outdated or duplicate of other changes that were pulled in many months ago. You likely need to rebase your branch
> 
> Rebased!

It still looks like there's 178 files being modified???

**Comment by skanda890** [comment]
But in GitHub Desktop it says that all commits have been rebased, I will try again and update.

**Comment by florelis** [comment]
Looks like you merged many of the branches we have open into yours?

image

I don't even know how you can clean that up short of creating a new branch based on `master` and cherry-picking only your changes (or re-doing them if they're small enough)

**Comment by skanda890** [comment]
@krschau, changes done.

**Pull request closed without merging**